src/main/resources/emojis.json

```diff
@@ -16802,5 +16802,14 @@
       "jolly_roger"
     ],
     "tags": []
+  },
+  {
+    "emojiChar": "🦾",
+    "emoji": "\uD83E\uDDBE",
+    "description": "machine arm",
+    "aliases": [
+      "machine arm"
```

Comment:
aliases are with underscore. According to https://emojipedia.org/mechanical-arm/, the correct alias would be `mechanical_arm`